### ISSUE TITLE
internationalisation

### ISSUE THREAD

**Issue by Dirk--** [open-boat-projects-org/jobs#2]
Most of our content is in German. Translation into English needs to be done ...

**Comment by Dirk--** [comment]
ich habe das plug-in 'TranslatePress - Multilingual' installiert. Scheint mir ganz passabel zu funktionieren:
- Deutsche Seite waehlen
- oben in der Leiste 'webseite uebersetzen'
- neuer Frame auf der linken Seite
- im Text einen Absatz waehlen (Symbol blauer Stift)
- im linken Frame englische Uebersetzung eingeben
- im linken Frame 'Uebersetzung speichern'
- zum Ausprobieren linken Frame schliessen

**Comment by wellenvogel** [comment]
geht das auch einfacher? Wäre ja wesnetlich schneller, gleich die ganze Seite in englisch zu schreiben...

**Comment by Dirk--** [comment]
ich habe vor allem mir das Leben einfach gemacht: 
Durch Nutzung des plugIns ist die Integrietaet aller Menues, Verlinkungen, plugIns etc. sichergestellt.
Die ganze wordpress-site haendisch mehrsprachig zu gestalten duerfte ein ziemlicher Aufwand sein ...
Es mag bessere plugIns geben, alle durchzuprobieren fehlt mir die Zeit.

**Comment by wellenvogel** [comment]
ok, habe mich durchgewürgt...

**Comment by wellenvogel** [comment]
ok- sehe gerade, das das nicht nur mein issue war - also wieder auf...

**Comment by Dirk--** [comment]
genau, es gibt noch einiges zu tun: Saskia hat ihre Hilfe angeboten, ich versuche mich ebenfalls an mehr Internationalitaet.
